aspnetcore/grpc/supported-platforms.md

```diff
@@ -44,10 +44,12 @@ gRPC services can be hosted on all built-in ASP.NET Core servers.
 >
 > * Kestrel
 > * TestServer
-> * IIS&dagger;
+> * IIS&Dagger;
 > * HTTP.sys&dagger;
 
-&dagger;IIS and HTTP.sys require .NET 5 and Windows 10 Build 20241 or later.
+&Dagger;IIS requires .NET 5 and Windows 10 Build 20241 or later.
+
+&dagger;HTTP.sys requires .NET 5 and Windows 10 Build 19529 or later.
```

Comment:
@Tratcher Is this version number right? I copied it from https://github.com/dotnet/aspnetcore/blob/c925f99cddac0df90ed0bc4a07ecda6b054a0b02/src/Servers/HttpSys/test/FunctionalTests/ResponseTrailersTests.cs#L58